packages/react-params/src/valueToText.tsx

```diff
@@ -53,6 +53,7 @@ export default function valueToText (type: string, value: any, swallowError = tr
             : value.toString()
           : (value instanceof Option) && value.isNone
             ? '<none>'
-            : JSON.stringify(value.toHuman())
+            // : JSON.stringify(value.toHuman())
+            : toFormattedBalance({ balance: new BN(value.toString()) })
```

Comment:
I have a feeling that you need to check the type of the value before using toFormattedBalance over the value..
The reason being not all the thing returned from storage is balance.. for example if you check  the first module
timestamp - and use the method now() -> it returns the current timestamp as "1,597,210,300,000" which we need not really format.. Attaching few examples
<img width="1181" alt="Screen Shot 2020-08-12 at 5 32 32 PM" src="https://user-images.githubusercontent.com/29415595/89978521-d34c1900-dcc1-11ea-8d0a-960cd18918c8.png">

Reply:
How about we focus on fixing balance formatting for now, and deal with relevant bugs as a following step, as many issues are already existing in the current ui app. I have added a task list at the bottom of this PR's description, and added this bug in that list. Once this PR is ok to go, I will work on `symbol` stuff as next, what do you think? @alexsednz  @holygits 

**Task list**:
- [x] Add `balance formatting` util https://github.com/cennznet/ui/pull/60 ;
- [ ] **Implement `balance formatting` in `Storage` page as a trial;**
- [ ] Implement `symbol` in `Storage` page;
- [ ] Implement `balance formatting` together with `symbol` in other pages;
- [ ] Ensure `balance formatting` is applied on balance data only;
- [ ] Remove legacy code;

Reply:
I would say we shouldn't create a bug in the first place. This doesn't seem to be the right place to call your function at all. I am even against checking the type. I would rather a solution that makes JSON.stringify(value.toHuman()) work for balance big numbers.

Reply:
@alexsednz  Nice idea, I would follow that approach to give it a try.

Reply:
@alexsednz  @KarishmaBothara I have added `type` checking condition and remove it as a bug from my todo list, please have a review again. cc @holygits

Reply:
Here is the page result, where we deal with `Balance` separately from other types.

<img width="522" alt="Screen Shot 2020-08-14 at 8 39 20 AM" src="https://user-images.githubusercontent.com/1513326/90184983-16b19f00-de0a-11ea-9ed4-3017f72207a2.png">